modules/xfem/src/userobjects/MeshCut2DRankTwoTensorNucleation.C

```diff
@@ -23,6 +24,15 @@ MeshCut2DRankTwoTensorNucleation::validParams()
   InputParameters params = MeshCut2DNucleationBase::validParams();
   params.addClassDescription(
       "Nucleate a crack in MeshCut2D UO based on a scalar extracted from a RankTwoTensor");
+  params.addRangeCheckedParam<Real>(
```

Comment:
I'm not sure if I like specifying this behavior in this way. What if instead we just have a boolean parameter that says whether to extend the nucleated crack to the nearest element edge? I would name it something like `extend_nucleated_cracks_across_elements` and make it default to 'true'.